packages/mdc-segmented-button/segmented-button/test/component.test.ts

```diff
@@ -58,13 +58,13 @@ const setupTest = () => {
 };
 
 const setAllSelected = (els: NodeListOf<Element>) => {
-  els.forEach((el) => {
+  [].slice.call(els).forEach((el: Element) => {
```

Comment:
We should be able to use Array#from now, jfyi!

Reply:
That's much better! Thanks..

Done.